robber/bad_expectation.py

```diff
@@ -4,6 +4,7 @@ class BadExpectation(Exception):
     """
 
     def __init__(self, message):
+        super(BadExpectation, self).__init__(message)
```

Comment:
Took me 5mins actually :| All I have to do was to search how to custom `Exception` in Python 2.